subprojects/build-init/src/main/groovy/org/gradle/api/tasks/wrapper/Wrapper.java

```diff
@@ -63,6 +64,7 @@
     private PathBase distributionBase = PathBase.GRADLE_USER_HOME;
     private String distributionUrl;
     private GradleVersion gradleVersion;
+    private String distType;
```

Comment:
But that's the point: As `BIN` is not accessible from `DistributionLocator` you'd need to write `getDistributionFor(version, "bin")`, which in turn is quite ugly. See https://github.com/sschuberth/gradle/commit/f22707e376d7cac41ef65c849b003507daf6d484.